recipes/pymc3-ext/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pymc3-ext" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymc3_ext-{{ version }}.tar.gz
+  sha256: 0e1be0d1c7e05f639d0cad5e6f064041b4952404e2dad29c4d65ef8e66c5409f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
Also it looks like `setuptools_scm` should be required under `host` since it's in the `setup_requires` in the `setup.py`

Reply:
Added!